pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[project]
+name = "nvda-mathcat"
+version = "20250706"
```

Comment:
Can you bump this version please (and merge in master, so this file doesn't appear in conflicts)